kickstarts/partials/post/gce.ks.erb

```diff
@@ -1,3 +1,18 @@
+# Install google compute engine packages
+# Disable functions that need network/retrieve metadata, as that will not be available in imagefactory installation
+cat >> /etc/default/instance_configs.cfg.template << EOF
+[InstanceSetup]
+network_enabled=false
+
+[NetworkInterfaces]
+setup=false
+EOF
+
+yum install -y google-compute-engine google-compute-engine-init google-config
+
+# Generate default cfg file on boot
```

Comment:
I think this should be a bit more clear.

What you're doing is wrapping the `yum install` in the config file that disables some settings, right?
The commit message explains it nicely, but I think this comment is a bit confusing, if for no other reason than it says "generate" when you're clearly removing something.